sql/core/src/test/scala/org/apache/spark/sql/execution/ui/AllExecutionsPageSuite.scala

```diff
@@ -73,10 +73,35 @@ class AllExecutionsPageSuite extends SharedSparkSession with BeforeAndAfter {
     map.put("failed.sort", Array("duration"))
     when(request.getParameterMap()).thenReturn(map)
     val html = renderSQLPage(request, tab, statusStore).toString().toLowerCase(Locale.ROOT)
-    assert(!html.contains("IllegalArgumentException"))
+    assert(!html.contains("illegalargumentexception"))
     assert(html.contains("duration"))
   }
 
+  test("SPARK-29453: correctly display tooltips for SQL Tab") {
```

Comment:
For the improvement PR, we don't use JIRA ID prefix. So, let's remove `SPARK-29453: ` from the test name.

Reply:
ok